operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -45,6 +45,8 @@ Entries should include a reference to the Pull Request that introduced the chang
 * [CHANGE] Nginx: uses the headless service of alertmanager, ingester and store-gateway as backends, because there are 3 separate services for each zone. #2778
 * [CHANGE] Gateway: uses the headless service of alertmanager as backend, because there are 3 separate services for each zone. #2778
 * [CHANGE] Update sizing plans (small.yaml, large.yaml, capped-small.yaml, capped-large.yaml). These reflect better how we recommend running Mimir and GEM in production. most plans have adjusted number of replicas and resource requirements. The only **breaking change** is in small.yaml which has reduced the number of ingesters from 4 to 3; for scaling down ingesters refer to [Scaling down ingesters](https://grafana.com/docs/mimir/latest/operators-guide/run-production-environment/scaling-out/#scaling-down-ingesters). #3035
+* [CHANGE] Change default securityContext of Mimir and GEM pods and containers so that they comply with a [Restricted pod security policy](https://kubernetes.io/docs/concepts/security/pod-security-standards/). This changes the user the containers run as from root to 10001. The files in the attached volumes to the pods should change ownership with the `fsGroup` change. Most CSI drivers support `fsGroup` or kubelet is able to do the ownership change instead of the CSI driver. This is not the case for the HostPath driver. If you are using HostPath or another driver that doesn't support `fsGroup`, then set the `securityContext` of all Mimir and GEM components to `{}` in your values file. If left as the default and `fsGroup` is not supported, components will fail to start. #3007
```

Comment:
```suggestion
* [CHANGE] Change default securityContext of Mimir and GEM Pods and containers, so that they comply with a [Restricted pod security policy](https://kubernetes.io/docs/concepts/security/pod-security-standards/). This changes what user the containers run as from `root` to `10001`. The files in the Pods' attached volumes should change ownership with the `fsGroup` change. Most CSI drivers support `fsGroup`, or kubelet is able to do the ownership change rather than the CSI driver. This is not the case for the HostPath driver; if you are using HostPath or another driver that doesn't support `fsGroup`, then set the `securityContext` of all Mimir and GEM components to `{}` in your values file. If left as the default and `fsGroup` is not supported, components will fail to start. #3007
```

Reply:
Vet "The files in the Pods' attached volumes should change ownership with the `fsGroup` change." for technical accuracy after my change. Also, what does _should_ mean here? Will they? If not, what happens? As a person, do I need to actively change them?